falcon/api.py

```diff
@@ -696,7 +696,7 @@ def _handle_exception(self, req, resp, ex, params):
         """
 
         for err_type, err_handler in self._error_handlers:
-            if isinstance(ex, err_type):
+            if isinstance(ex, tuple(err_type)):
```

Comment:
I think it would be better to just normalize to a tuple up front when the error handler is added.

Reply:
An idea how to implement that part, one way could be something like
```python
# ...
if not issubclass(exc_type, BaseException):  # not an exception type, assume iterable
    exc_type = tuple(exc_type)
```

Reply:
thanks for the suggestions guys! Sorry I started the work but then dropped it due to lack of time....I will soon get back on it!